web/assets/css/_vars.less

```diff
@@ -32,4 +31,71 @@
 @yellow: #f1c40f;
 @purple: #5c3bd1;
 
+/* the new style colours below */
+
+@black: #000000;
+@white: #FFFFFF; 
+
+@grey10: #F2F2F2;
+@grey20: #E2E2E2;
+@grey30: #C6C6C6;
+@grey40: #A4A4A4;
+@grey50: #808080;
+@grey60: #666666;
+@grey70: #4D4D4D;
+@grey80: #363636;
+@grey90: #262626;
+@grey100: #151515;
+@success10: #DFF7EA;
+@success20: #9AF5C2;
+@success30: #5AE095;
+@success40: #1CBD63;
+@success50: #0D9448;
+@success60: #0B7539;
+@success70: #0F5930;
+@success80: #0D3D22;
+@success90: #0F2B1C;
+@success100: #0B1710;
+@paused10: #E8F3FA;
+@paused20: #C4E7FF;
+@paused30: #87CFFF;
+@paused40: #4BAFF2;
+@paused50: #1586D1;
+@paused60: #0969AB;
+@paused70: #075082;
+@paused80: #083757;
+@paused90: #0B2638;
+@paused100: #06151F;
+@failure10: #FAEFED;
+@failure20: #FFD9D4;
+@failure30: #F7B6AD;
+@failure40: #EB8E81;
+@failure50: #DB5442;
+@failure60: #BD311E;
+@failure70: #912617;
+@failure80: #5E1D14;
+@failure90: #3B1915;
+@failure100: #1F110F;
+@pinned10: #F3F0FC;
+@pinned20: #E5DEFA;
+@pinned30: #CDC0FA;
+@pinned40: #AF99FF;
+@pinned50: #8866FF;
+@pinned60: #6948E0;
+@pinned70: #5035B0;
+@pinned80: #38296E;
+@pinned90: #271E4A;
+@pinned100: #161029;
+@error10: #FAF0E1;
+@error20: #FCE0B1;
+@error30: #F2BF6B;
+@error40: #DE951D;
+@error50: #B57200;
+@error60: #915C00;
+@error70: #6E4500;
+@error80: #4A3107;
+@error90: #30220A;
+@error100: #191205;
+
```

Comment:
Sanity check: do folks like having 10 be the lightest tint and 100 be the darkest shade, or would the reverse make more sense? Different products do this in different ways.

Reply:
10 being darkest is more intuitive to me, but seems like it'd be pretty painful to reverse so I wouldn't worry about it too much 😁

Reply:
i can convince myself that both ways work fine... 😅 

i can also convince myself that the scale should be grey0 - grey200, where grey100 is pure grey, 🤔 (i don't think we should pick this method)

Reply:
Google, Atlassian, Lyft, IBM, Microsoft, and a few others I've looked at all seem to use higher numbers to signify darker colours, while others I looked at use novel systems (like 'lightest, light, neutral, dark, darkest', or 'blue-light-20, blue-light-10, blue, blue-dark-10, blue-dark-20). I haven't found anyone who uses '10' for dark and '100' for light, despite the fact that it would correlate to brightness values to do so.

Let's stick to convention for now, we can adjust with some find-and-replace magic in the future if we need to.